pkg/types/validation/installconfig_test.go

```diff
@@ -451,6 +456,44 @@ func TestValidateInstallConfig(t *testing.T) {
 			}(),
 			expectedError: `^platform\.vsphere.vCenter: Required value: must specify the name of the vCenter$`,
 		},
+		{
+			name: "empty proxy settings",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Proxy.HTTPProxy = ""
+				c.Proxy.HTTPSProxy = ""
+				c.Proxy.NoProxy = ""
+				return c
+			}(),
+			expectedError: `^proxy: Required value: Must include at least one of httpProxy, httpsProxy, or noProxy$`,
+		},
+		{
+			name: "invalid HTTPProxy",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Proxy.HTTPProxy = "bad-uri"
+				return c
+			}(),
+			expectedError: `^HTTPProxy: Invalid value: "bad-uri": invalid URI "bad-uri" \(no scheme\)$`,
+		},
+		{
+			name: "invalid HTTPSProxy",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Proxy.HTTPSProxy = "bad-uri"
+				return c
+			}(),
+			expectedError: `^HTTPSProxy: Invalid value: "bad-uri": invalid URI "bad-uri" \(no scheme\)$`,
+		},
+		{
+			name: "invalid NoProxy",
```

Comment:
maybe add cases like:
- noProxy invalid domain
- noProxy invalid cidr
- noProxy one of many invalid
- noProxy two of many invalid